ormar/fields/model_fields.py

```diff
@@ -68,7 +68,8 @@ def convert_choices_if_needed(
     nullable: bool,
     scale: int = None,
     represent_as_str: bool = False,
-) -> Set:
+    sort_choices: bool = True,
+) -> List:
```

Comment:
This should typecheck since `BaseField` types choices as a Sequence instead of a concrete type like list or set